build.gradle

```
@@ -21,12 +22,21 @@ configurations {
 
 repositories {
 	mavenCentral()
+//	// Spring Milestone 저장소 추가
+//	maven { url 'https://repo.spring.io/milestone' }
+//	// Spring Snapshot 저장소 추가
+//	maven { url 'https://repo.spring.io/snapshot' }
 }
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+
+	// Spring AI Vertex AI Gemini
+	implementation platform('org.springframework.ai:spring-ai-bom:1.0.0')
```

P2
어 이거 이거 이용해서 정의해 주시면 안 되나여
```
ext {
    springAiVersion = "1.0.0"
}
```
```suggestion
    implementation platform("org.springframework.ai:spring-ai-bom:$springAiVersion")